### DISCUSSION TITLE
Windows 11 can't display icons?

### DISCUSSION THREAD

**Discussion by aunsparrow** [wailsapp/wails#3752] [Q&A]
windows 10
![image](https://github.com/user-attachments/assets/403a7763-683d-40ff-940b-bc9f3110aa6f)

windows 11
![image](https://github.com/user-attachments/assets/0f66f5f7-3802-46a4-b870-27888743266c)

**Comment by ilaziness** [comment]
windows 11
My normal, no problem.

**Comment by leaanthony** [comment]
This can sometimes happen due to windows thumbnail cache
